src/box/lua/config/configdata.lua

```diff
@@ -411,6 +411,38 @@ local function validate_failover(found, peers, failover, leader)
                 leader, found.replicaset_name, found.group_name, leader), 0)
         end
     end
+
+    -- Verify that 'election_mode' option is set only in the
+    -- 'failover: election' mode.
+    --
+    -- The alternative would be silent ignoring the election
+    -- mode if failover mode is not 'election'.
+    --
+    -- For a while, a simple and straightforward approach is
+    -- chosen: let the user create an explicit consistent
+    -- configuration manually.
+    --
+    -- We can relax it in a future, though. For example, if two
+    -- conflicting options are set in different scopes, we can
+    -- ignore one from the outer scope.
+    if failover ~= 'election' then
+        for peer_name, peer in pairs(peers) do
+            local election_mode = instance_config:get(peer.iconfig_def,
+                'replication.election_mode')
+            if election_mode ~= nil then
```

Comment:
It should be `if election_mode ~= "off" then ...`. Otherwise you contradict with the commit message and changelog, now it's forbidden to specify `election_mode`  at all.

Reply:
Ouch. My mistake, sorry. Will fix.